src/ngraph/runtime/backend.hpp

```diff
@@ -48,6 +49,10 @@ class ngraph::runtime::Backend
     ///   does not exist.
     static std::unique_ptr<Backend> create(const std::string& type);
 
+    /// \brief Get handle to backend shared library
+    /// \returns backend handle
+    static DL_HANDLE get_handle(const std::string& type);
```

Comment:
If the backend is not found then how is the error propagated? This function should behave exactly the same way as the create_backend(). 

Better yet - consider using the ngraph::Backend* as the key to the registry - not the name. That way - the caller needs to make the "create_backend()" first to get the backend created - and then get its handle. Then it's failsafe.